### DISCUSSION TITLE
[Linux fedora] Installation / dependency error with Webui already installed.

### DISCUSSION THREAD

**Discussion by errchh** [Comfy-Org/ComfyUI#142] [Q&A]
Hi, sorry for a newbie question. I have ran into an error during the installation. I have already installed webui. 

Is it possible to have both UI installed, with all the dependencies separated without trouble from each other? 

Here is my error message in terminal: 

> Installing collected packages: protobuf
>   Attempting uninstall: protobuf
>     Found existing installation: protobuf 3.19.6
>     Uninstalling protobuf-3.19.6:
>       Successfully uninstalled protobuf-3.19.6
> ERROR: pip's dependency resolver does not currently take into account all the packages that are installed. This behaviour is the source of the following dependency conflicts.
> tensorflow 2.11.0 requires protobuf<3.20,>=3.9.2, but you have protobuf 3.20.0 which is incompatible.
> Successfully installed protobuf-3.20.0
> [me@fedora]$

**Comment by comfyanonymous** [comment]
I get this error on colab and it works anyways. You should just try running it and it's probably going to work.

**Reply by errchh** [reply]
I just stopped when I saw that error msg. I got to give it a go again. // great modern UI btw. // thank you!!
